lib/github-glue.ts

```diff
@@ -100,21 +101,52 @@ export class GitHubGlue {
      */
     public async addPRCc(pullRequestURL: string, cc: string):
         Promise<void> {
-        const id = cc.match(/(<.*>)/);
+        const id = cc.match(/<(.*)>/);
 
         if (!id || id[1].match(/gitster@pobox\.com/)) {
```

Comment:
I guess at this stage it would make sense to change this to `id[1] === "gitster@pobox.com"`?

Reply:
Agreed.

Reply:
Sure.  I had intended to do it and forgot in the morning.